src/components/SideSheet/Footer/Footer.tsx

```diff
@@ -11,7 +11,7 @@ type TProps = {
     confirm: TButtonPropTypes
     extraButton?: TButtonPropTypes
   }
-  onClose: () => void
+  onClose: (e: TClickEventType | undefined) => void
```

Comment:
@rispiryan jan I think you can write like this
`onClose: (e?: TClickEventType) => void`